<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="DebugScreenGroup">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>381</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="DebugLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>361</width>
       <height>20</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LayoutDirection::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Debug picture</string>
     </property>
     <property name="textFormat">
      <enum>Qt::TextFormat::PlainText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="DebugPicLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>29</y>
       <width>361</width>
       <height>41</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>361</width>
       <height>41</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="DebugNoteLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>361</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Note: picture in debug screen must be like on a sample picture.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="EventHystoryBox">
    <property name="geometry">
     <rect>
      <x>399</x>
      <y>9</y>
      <width>391</width>
      <height>571</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPlainTextEdit" name="EventHistoryTextEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>371</width>
       <height>551</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="ButtonPanel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>381</width>
      <height>461</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="SampleButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>10</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Sample</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ChangeCoordsButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Change coords</string>
     </property>
    </widget>
    <widget class="QPushButton" name="StartButton">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>100</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="StopButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
    <widget class="QPushButton" name="GoBackButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>400</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Go back</string>
     </property>
    </widget>
    <widget class="QPushButton" name="EvemtBoxOnTop">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>400</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Event box on top</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="enabled">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>StartButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>StopButton</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>315</x>
     <y>240</y>
    </hint>
    <hint type="destinationlabel">
     <x>80</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>StartButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>StartButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>315</x>
     <y>240</y>
    </hint>
    <hint type="destinationlabel">
     <x>315</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>StopButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>StopButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>240</y>
    </hint>
    <hint type="destinationlabel">
     <x>80</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>StopButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>StartButton</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>240</y>
    </hint>
    <hint type="destinationlabel">
     <x>315</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
